zoomx/src/main/java/com/zoomx/zoomx/config/Config.kt

```diff
@@ -0,0 +1,47 @@
+package com.zoomx.zoomx.config
+
+import android.content.Context
+
+/**
+ * Created by Ahmed Fathallah on 12/11/2017.
+ */
+
+class Config internal constructor(builder: Builder) {
```

Comment:
you really will not need a builder pattern with Kotlin, you could just use (apply, with, ... etc) functional methods